packages/admin/cms-admin/src/common/header/UserHeaderItem.tsx

```diff
@@ -48,8 +62,13 @@ export function UserHeaderItem(props: UserHeaderItemProps): React.ReactElement {
     const [showAboutModal, setShowAboutModal] = React.useState(false);
     const [signOut, { loading: isSigningOut }] = useMutation<GQLSignOutMutation>(signOutMutation);
 
+    const AccountIcon = () => (user.impersonated ? <Account color="info" /> : <Account />);
```

Comment:
```suggestion
    const accountIcon = user.impersonated ? <Account color="info" /> : <Account />;
```

No need for an component here